arguebuf/schema/sadface.py

```diff
@@ -0,0 +1,38 @@
+import typing as t
+import arguebuf as ag
+
+NodeType = t.Literal["scheme", "atom"]
+DATE_FORMAT = "YYYY-MM-DD HH:mm:ss"
+
+
+class Node(t.TypedDict):
+    id: str
+    text: str
+    type: NodeType
+    name: str
+    sources: list
+    metadata: t.Dict[str, t.Any]
+
+
+class Edge(t.TypedDict):
+    id: str
+    source_id: str
+    target_id: str
+
+
+class Core(t.TypedDict):
+    analyst_email: str
+    analyst_name: str
+    created: str
+    description: str
+    edited: str
+    id: str
+    notes: str
+    title: str
+    version: str
+
+
+class Graph(t.TypedDict):
+    nodes: t.List[Node]
+    edges: t.List[Edge]
+    metadata: Core
```

Comment:
Not quite right. The attribute `core` is [a child of the metadata object](https://github.com/Open-Argumentation/SADFace/blob/c17fd89e3f6355cbe9bc4bd320bb1461894e2752/examples/hangback/data.json#L30)